codec/H264Header.h

```diff
@@ -0,0 +1,64 @@
+#pragma once
+
+#include <arpa/inet.h>
+#include <cassert>
+#include <cstddef>
+#include <cstdint>
+
+namespace codec
+{
+
+namespace H264Header
+{
+
+constexpr uint8_t getNalUnitType(const uint8_t nalHeader)
+{
+    return nalHeader & 0x1f;
+}
+
+constexpr uint8_t getFuNalStartBit(const uint8_t fuHeader)
+{
+    return fuHeader & 0x80;
+}
+
+inline bool isKeyFrameStapA(const uint8_t* payload, const size_t payloadSize)
+{
+    size_t offset = 1;
+
+    while (payloadSize - offset >= 3)
```

Comment:
There is no check that the payloadSize is > 0 in callstack to this method. I would prefer if this method return false if ever (payloadSize < offset);

Reply:
Thanks, I fixed the incorrect length check + added a test for malformed stap-a packet. I also inlined the function to make it clear it is not meant to be called separately.